src/containerapp/azext_containerapp/custom.py

```diff
@@ -113,7 +117,7 @@
 
 
 logger = get_logger(__name__)
-
+DEFAULT_CONFIGURED_STR = 'Configured'
```

Comment:
add it in _constants file?

Reply:
DEFAULT_CONFIGURED_STR = 'Configured' removed totally